CHANGELOG.md

```diff
@@ -1,16 +1,34 @@
 ## 2.6.0 (Unreleased)
 
+FEATURES:
+
+* **New Data Source:** `vcd_vapp_vm` VM - [GH-218]
+
+BUG FIXES:
+
+* Removed `power_on` property from data source `vcd_vapp`, as it is a directive used during vApp build.
```

Comment:
This statement doesn't match UI behavior 
![image](https://user-images.githubusercontent.com/41415459/68301254-37dda600-00a8-11ea-9ea3-98f87bd92381.png)

Could you explain us what is happening here

Reply:
This has been explained in the vApp PR.
`power_on` is not a reliable state. It is defined as such when state code = 4. However, this code can have several values, which may or may not be compatible with the state of being powered on. 
For this reason, the property `power_on` is used as a directive, but we retrieve the naked status code and status text as separate fields.

The UI gets such status and has some internal code to determine when a status should be considered ON. According to the API, only status n. 4 is "ON", but this field is overloaded with several more meanings that have nothing to do with being on or not.

Reply:
My idea come from diff point of view  - didn't this field allowed to initiate power one or power off?

Reply:
It does. It is used in the resource, but not fetched when we read for the reason stated above. Hence, it is not needed in the **data source**

Reply:
oh yes, this is only datasource. thank you for explanation